.env.example

```
@@ -1,10 +1,66 @@
-GITHUB_API_KEY="your_github_api_key_here"             # Optional: For GitHub import/export features. Format: ghp_... or github_pat_...
\ No newline at end of file
+# API Keys Configuration
+# Copy this to .env.local or environment-specific files
+# DO NOT commit this file to git
+
+# ===================================
+# OpenRouter API (Primary for AI operations)
+# ===================================
+# Required for: Transaction categorization, document analysis, insights generation
+# Get your key at: https://openrouter.ai/keys
+ANTHROPIC_API_KEY="sk-or-v1-xxxx-xxxx-xxxx-xxxx"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)